cmd/checkpoint/main.go

```diff
@@ -46,12 +50,32 @@ var tempAPIServerManifest = v1.Pod{
 	},
 }
 
+var tempEtcdManifest = v1.Pod{
+	TypeMeta: unversioned.TypeMeta{
+		APIVersion: "v1",
+		Kind:       "Pod",
+	},
+	ObjectMeta: v1.ObjectMeta{
+		Name:      tempEtcd,
+		Namespace: api.NamespaceSystem,
+	},
+}
+
 var tempPodSpecMap = map[string]v1.Pod{
 	tempAPIServer: tempAPIServerManifest,
+	tempEtcd:      tempEtcdManifest,
 }
 
 func main() {
 	glog.Info("begin pods checkpointing...")
+	etcd := flag.Bool("experimental-self-hosted-etcd", false, "Checkpoint self hosted etcd.")
```

Comment:
If that is the case, I think it makes sense to finish the annotation thing first? Then the checkpointer knows what to pick up to checkpoint.